ts/screens/modal/components/IdentificationNumberPad.tsx

```diff
@@ -1,13 +1,13 @@
-import * as React from "react";
-import { useState, useCallback } from "react";
 import {
+  BiometricsValidType,
   CodeInput,
-  NumberPad,
-  VSpacer,
   IOStyles,
-  BiometricsValidType,
-  IconButton
+  IconButtonContained,
+  NumberPad,
+  VSpacer
 } from "@pagopa/io-app-design-system";
+import * as React from "react";
+import { useCallback, useState } from "react";
 import { View } from "react-native";
 import I18n from "../../../i18n";
 import { isDevEnv } from "../../../utils/environment";
```

Comment:
Same as above. Too much entropy... I suggest to disable this kind of plugin. What do you think?